app/controllers/railway_stations_controller.rb

```diff
@@ -35,6 +35,14 @@ def destroy
     redirect_to railway_stations_path, notice: 'Railway station was successfully destroyed.'
   end
 
+  def update_station
+    @route = Route.find(params[:route_id])
```

Comment:
https://dev.to/mikamai/effective-service-objects-in-ruby-2ga0#service-composition